### ISSUE TITLE
"Lightweight Node REPL for iOS and Android"

### ISSUE THREAD

**Issue by brentvatne** [mzabriskie/PocketNode#3]
Not exactly a Node REPL because JavaScriptCore doesn't include node standard library. It's more of a Lightweight JavaScriptCore REPL :)

**Comment by mzabriskie** [comment]
Yes, technically you're correct. PocketNode had a better ring to it than PocketJavaScriptCore :-)

**Comment by brentvatne** [comment]
hahaha that is very true
